src/test/java/com/ays/backend/user/service/UserServiceTest.java

```diff
@@ -48,4 +60,116 @@ void shouldCreateUser() {
         assertEquals(signUpRequest.getLineNumber(), savedUserDTO.getLineNumber());
         assertEquals(signUpRequest.getStatusId(), savedUserDTO.getUserStatus().ordinal());
     }
+
+    @Test
+    public void shouldGetAllUsers() {
+
+        // given
+        User userInfo1 = User.builder()
+                .username("username 1")
+                .firstName("firstname 1")
+                .lastName("lastname 1")
+                .userRole(UserRole.ROLE_VOLUNTEER)
+                .build();
+
+        User userInfo2 = User.builder()
+                .username("username 2")
+                .firstName("firstname 2")
+                .lastName("lastname 2")
+                .userRole(UserRole.ROLE_VOLUNTEER)
+                .build();
+
+        User user1 = new UserBuilder()
+                .withAllInfo(userInfo1)
+                .build();
+
+        User user2 = new UserBuilder()
+                .withAllInfo(userInfo2)
+                .build();
+
+
+        UserDTO userDTO1Info = UserDTO.builder()
+                .username("username 1")
+                .firstName("firstname 1")
+                .lastName("lastname 1")
+                .userRole(UserRole.ROLE_VOLUNTEER)
+                .build();
+
+        UserDTO userDTO2Info = UserDTO.builder()
+                .username("username 2")
+                .firstName("firstname 2")
+                .lastName("lastname 2")
+                .userRole(UserRole.ROLE_VOLUNTEER)
+                .build();
+
+        UserDTO userDTO1 = new UserDTOBuilder()
+                .withAllInfo(userDTO1Info)
+                .build();
+
+        UserDTO userDTO2 = new UserDTOBuilder()
+                .withAllInfo(userDTO2Info)
+                .build();
+
+
+        // Create a Page object with the sample users
+        Page<User> users = new PageImpl<>(Arrays.asList(user1, user2));
```

Comment:
We can create and return lists of `UserDTOs` inside the `UserDTOBuilder` class, so that they can be used when needed.